frontend/src/components/Dropdown/Dropdown.styled.js

```diff
@@ -0,0 +1,80 @@
+import styled from "styled-components";
+import {device} from "../../RWD/device";
+
+export const StyledDropdown = styled.div`
+  min-width: 160px;
+  padding: 0.5rem;
+  position: absolute;
+  top: calc(0% + .7rem);
+  right: 0;
+  background-color: #fff;
+  border: 1px solid #e1e1e1;
+  border-radius: 5px;
+  overflow: hidden;
```

Comment:
```suggestion
  position: absolute;
  top: calc(0% + .7rem);
  right: 0;
  min-width: 160px;
  padding: 0.5rem;
  background-color: #fff;
  border: 1px solid #e1e1e1;
  border-radius: 5px;
  overflow: hidden;
```

Naszym zbiorem jest position, więc szerokości są pod position.